<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="task_4">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Задание № 4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="task_7">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Задание № 7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="task_9">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Задание № 9</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="task_10">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Задание № 10</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="task_11">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Задание № 11</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="task_12">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Задание № 12</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="task_13">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Задание № 13</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="task_14">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Задание № 14</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="task_15">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Задание № 15</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="title_label">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Тренажёр для подготовки к ЕГЭ по русскому языку</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Дополнительные возможности</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Выгрузить статистику</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Обнулить статистику</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Добавить вариант задания</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Удалить вариант задания</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Обратная связь</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
